UI/RemoteISOScreen.cpp

```diff
@@ -268,19 +279,34 @@ static bool LoadGameList(const std::string &host, int port, std::vector<std::str
 	std::vector<std::string> items;
 	result.TakeAll(&listing);
 
-	SplitString(listing, '\n', items);
-	for (const std::string &item : items) {
-		if (!endsWithNoCase(item, ".cso") && !endsWithNoCase(item, ".iso") && !endsWithNoCase(item, ".pbp")) {
-			continue;
+	if (startsWith(responseHeaders[0],"Server: PPSSPPServer") || startsWith(responseHeaders[0], "Server: SuperDuperServer")) {
```

Comment:
The correct way to handle this would be with a Content-Type (it would be `text/plain` from PPSSPP.)  This would allow other clients to send the same responses without pretending to be PPSSPPServer.

-[Unknown]